deflect/StreamEventThread.cpp

```diff
@@ -1,200 +0,0 @@
-/*********************************************************************/
```

Comment:
? haven't seen this file ever? wow

Reply:
I found it by accident, it had been forgotten since 2013...!